pkg/api/types/global.go

```diff
@@ -0,0 +1,21 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package types
+
+import "github.com/containerd/nerdctl/pkg/config"
+
+type GlobalCommandOptions config.Config
```

Comment:
Do we need this?

Reply:
If we introduce a config object into XXCommandOptions object, it will make people confused.

So, I think using the typing alias may be an excellent way to keep the code semantic.